src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/Main.java

```diff
@@ -126,13 +133,22 @@ private static void processFile(final File inputFile, final File outputFile) thr
                     "that was submitted to APIView was named " + inputFile.getName()));
         }
 
-        // Write out to the filesystem
-        try (JsonWriter jsonWriter = JsonProviders.createWriter(Files.newBufferedWriter(outputFile.toPath()))) {
-            apiListing.toJson(jsonWriter);
+        try {
+            // Write out to the filesystem, make the file if it doesn't exist
+            if (!outputFile.exists()) {
+                if (!outputFile.createNewFile()) {
+                    System.out.printf("Failed to create output file %s%n", outputFile);
+                }
+            }
+            try (JsonWriter jsonWriter = JsonProviders.createWriter(Files.newBufferedWriter(outputFile.toPath()))) {
+                apiListing.toJson(jsonWriter);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Why do we choose to print the exception stack trace instead of failing the run?

Reply:
Diagnostics - also, soon APIView will start to show the output from the parser, so we can start to diagnose issues with generating reviews from within the UI.